awesome.c

```diff
@@ -559,13 +561,14 @@ exit_help(int exit_code)
     FILE *outfile = (exit_code == EXIT_SUCCESS) ? stdout : stderr;
     fprintf(outfile,
 "Usage: awesome [OPTION]\n\
-  -h, --help             show help\n\
-  -v, --version          show version\n\
-  -c, --config FILE      configuration file to use\n\
-      --search DIR       add a directory to the library search path\n\
-  -k, --check            check configuration file syntax\n\
-  -a, --no-argb          disable client transparency support\n\
-  -r, --replace          replace an existing window manager\n");
+  -h, --help           show help\n\
+  -v, --version        show version\n\
+  -c, --config FILE    configuration file to use\n\
+      --search DIR     add a directory to the library search path\n\
+  -k, --check          check configuration file syntax\n\
+  -a, --no-argb        disable client transparency support\n\
+  -m, --screen on|off  enable or disable automatic screen creation (default: on)\n\
```

Comment:
`--screen off` really is not self-explanatory. How about `--auto-screen`? Or inverting this and calling it `--manual-screen` so that `-m` makes more sense?

Reply:
https://github.com/awesomeWM/awesome/pull/2790/files/e69d1493f21e6d0b65d13a4c4c9b252ece486589#diff-5bb7c29334cbe6889621399f4daa8bc5

Please argue with @blueyed about the name. What you propose is what I originally did.